### ISSUE TITLE
Reading: Improving Temporal Generalization of Pre-trained Language Models with Lexical Semantic Change

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#241]
## 0. Paper
- paper: [arxiv](https://arxiv.org/abs/2210.17127)
- authors: Zhaochen Su, Zecheng Tang, Xinyan Guan, Juntao Li, Lijun Wu, Min Zhang

## 1. What is it?
They reveal that semantically different words/tokens of pre-trained language models improve performance.

## 2. What is amazing compared to previous works?
They proposed a fine-tuning method of pre-trained language models (PLMs) using misalignment (semantically different) of words/tokens across different time spans or topics.

![スクリーンショット 2022-11-01 15 15 06](https://user-images.githubusercontent.com/45454055/199268225-01adeff4-2551-4346-adec-30e975d90d03.png)

## 3. Where is the key to technologies and techniques?
1. obtain target words (TF-iDF)
2. clustering (for each word, usage vectors are obtained from pre-trained BERT)
3. calculate misalignment (change) score for each word (JSD)
4. MASK random or misalignment words

### 4.1 Difficulty of misalignment words
![スクリーンショット 2022-11-01 15 29 59](https://user-images.githubusercontent.com/45454055/199272246-dc34ea1c-0775-468b-8e76-4fc9ae7de99b.png)

Figure 2 shows that masking misalignment words (lmlm) is more difficult than baselines.

### 4.2 Main result(s)
![スクリーンショット 2022-11-01 15 37 07](https://user-images.githubusercontent.com/45454055/199273856-b7e16e02-603a-42f9-80e7-09dea0d617ba.png)

Table 2 shows that their method (+lmlm) improve performance.
